app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt

```diff
@@ -32,11 +34,15 @@ import org.fossasia.openevent.general.utils.Utils.progressDialog
 import org.fossasia.openevent.general.utils.extensions.nonNull
 import org.jetbrains.anko.design.longSnackbar
 import org.jetbrains.anko.design.snackbar
+import org.koin.androidx.viewmodel.ext.android.sharedViewModel
 import org.koin.androidx.viewmodel.ext.android.viewModel
 
+const val PLAY_STORE_BUILD_FLAVOR = "playStore"
```

Comment:
Declare it once. It is public value, you can use it in different class also.

Reply:
updated. will squash once you approve